lib/rex/ui/text/shell.rb

```diff
@@ -219,7 +219,7 @@ def print_error(msg='')
 
     self.on_print_proc.call(msg) if self.on_print_proc
     # Errors are not subject to disabled output
-    log_output(output.print_error(msg))
+    log_error(output.print_error(msg))
```

Comment:
Logging just error messages won't capture the whole story, lots of errors occur outside the shell that we want to capture. I think adding a thing to `elog` to also send everything to the `error` log sink would be a good thing, possibly in addition to this. Personally, I would prefer the cases you are intending to catch here (stack traces and such) also start being `elog`'d, but that may be debatable. Maybe keep a ring buffer of all output in memory for use with `Debug` to get the best of both worlds?

Reply:
Have altered elog to log all messages in both the specified sink and the error sink. If the error sink is the specified sink, the message is only logged once in the error sink. 

Also `log_error` already sends the strings it gets passed to elog, are there any possible issues with this?